.github/workflows/wheels.yml

```diff
@@ -85,10 +85,9 @@ jobs:
           - [ubuntu-22.04-arm, manylinux_aarch64, ""]
           - [ubuntu-22.04-arm, musllinux_aarch64, ""]
           - [macos-13, macosx_x86_64, openblas]
-
-          # targeting macos >= 14. Could probably build on macos-14, but it would be a cross-compile
           - [macos-13, macosx_x86_64, accelerate]
-          - [macos-14, macosx_arm64, accelerate]  # always use accelerate
+          - [macos-14, macosx_arm64, openblas]
```

Comment:
I suspect the wheel won't be installable on macos11/12/13 with this.

Reply:
Do I need to give it an explicit `MACOSX_DEPLOYMENT_TARGET=11.0`?

Reply:
The build creates a `numpy-2.4.0.dev0-cp312-cp312-macosx_11_0_arm64.whl` wheel, without needing to specify `MACOSX_DEPLOYMENT_TARGET`. Is this incorrect?

Reply:
It's not clear to me how the 11_0 appears there though. In the cirrus job it's specified in the configuration. 
It's probably worth experimenting with `otool -l somerandom.dylib`, there should be something specified in the output of that. I'm basing this off https://stackoverflow.com/questions/17461196/determining-os-x-sdk-and-deployment-target-versions-from-framework, although that's quite old.

Reply:
I think it comes from cibuildwheel [here](https://github.com/pypa/cibuildwheel/blob/db2d59be22f774aea1a52534918bd12b3d370965/cibuildwheel/platforms/macos.py#L288)

Reply:
This should be fine I'd think indeed; `cibuildwheel` sets its own default for each platform if we don't set a different one explicitly.